docsource/modules150-160.rst

```diff
@@ -854,7 +854,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | website_forum                                   |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| website_google_map                              |Nothing to do         |                                                 |
+| website_google_map                              | Nothing to do        |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | website_hr_recruitment                          |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
```

Comment:
Please do not modify other line than the one related to the migration of the module `website_event_questions`. It may generate conflict if several people do that.

Reply:
ok :grimacing: 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWFtcnEzOG14MmMwaGI3cG92YWZiNmdvN3llhnNGkxYzMzc3Y0NiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/PVyzIA9wkvbv3mXkQ7/giphy.gif)

Reply:
done